pom.xml

```diff
@@ -94,8 +84,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.3.2</version>
                 <configuration>
-                    <source>1.5</source>
-                    <target>1.5</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
                 </configuration>
             </plugin>
             <plugin>
```

Comment:
remove extra log4j dependency, there are two dependencies added, one for log4j and one for slf4j